### ISSUE TITLE
Add possibility to exclude a genre in the Movies and Series view

### ISSUE THREAD

**Issue by f18m** [seerr-team/seerr#1125]
### Description

Let's say you want to view all movies that are either crime, science-fiction or adventure genres.
Today the "Genre" filter is an AND filter and placing all those 3 filters together is not what you mean.

### Desired Behavior

I would like the ability to put multiple genre in OR condition or, alternatively or in a complementary way, the possibility to exclude a particular genre from the results

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by BillTheGoat** [comment]
Second this. I rarely watch animation and would like to be able to exclude this from results.

**Comment by BenH4** [comment]
Would be awesome! For some inspiration, one website filtering that I love to use is the one on the betaseries website where by clicking twice the genre, it excludes it from the search. Plus it is possible to filter by release date and more.
[https://www.betaseries.com/en/movies/other-new](https://www.betaseries.com/en/movies/other-new)

**Comment by znidaxp** [comment]
Yes, please implement this. It can be global filter or per-user filter (better this way). Let's say i want to exclude "anime", there should be an option to exclude certain tags.
